frontend/src/components/Playback/UpNext.vue

```diff
@@ -67,75 +67,57 @@
   </v-container>
 </template>
 
-<script lang="ts">
-import { defineComponent } from 'vue';
+<script setup lang="ts">
```

Comment:
@ferferga @ThibaultNocchi I didn't see this component referenced anywhere. Is this something that will be reconnected at some point or can it be discarded?

Reply:
Adding this to my list of future TODOs, but excluding it from the scope of this PR